docs/usermanual.md

```diff
@@ -85,6 +85,8 @@ following query parameters:
 -   **k (API Key)**: API Key for the service the device is registered on.
 -   **t (timestamp)**: Timestamp of the measure. Will override the automatic IoTAgent timestamp (optional).
 
+=======
+
```

Comment:
```suggestion
```

Seems to be the leftover of some conflict resolution operation.

Reply:
> Seems to be the leftover of some conflict resolution operation.

@fgalan 
I've incorporated review comments. If it is OK then please merge the PR in master branch.